### DISCUSSION TITLE
唐鳳接任數位發展部召集人！掌管電信、資安，新設部門是什麼一次看懂

### DISCUSSION THREAD

**Discussion by Joan0518** [1101-GS3073-C/Python-Community-GS3073-C#404] [科技新知]
也分享一下數位發展部的相關新聞
數位發展部有規劃較詳細的施政內容了，雖然看起來還是很模糊，再看看未來的作為吧，可以點進網址看看
原本數位發展部首任部長呼聲最高的是郭耀煌與政委唐鳳，但前者五月底辭了政務委員，重回成大當教授了~~
[新聞內容](https://www.bnext.com.tw/article/61954/ey-gov-0325)
![img-1616645379-37150@900](https://user-images.githubusercontent.com/90574174/173426820-afcaadf5-c3db-4afb-9a2b-693502f82480.jpg)
![img-1616645383-96759@900](https://user-images.githubusercontent.com/90574174/173428001-452aadd8-be0f-4f1c-9309-bb6b54530779.jpg)
![img-1616645387-48938@900](https://user-images.githubusercontent.com/90574174/173428043-906ba6aa-7520-47b5-a519-baceabcc78f2.jpg)

**Comment by AngusGuei** [comment]
希望可以杜絕像n號房事件這種低劣行為😮‍💨

**Comment by luo910227** [comment]
希望在唐鳳帶領下資訊安全能夠更加受到保障，這一直是台灣需要面對的問題，尤其從對岸來的攻擊。

**Comment by emily40533** [comment]
雖然唐鳳因為其獨特的經歷以及自身魅力（還有迷因？）而知名度飆高，也成為呼聲很高的部長候選人，但我記得被任命的前後，他都在人事上遇到了不少問題。
樓主的這篇新聞已經是三月的內容了，可能沒有follow到一些後續。像我記得前政委郭耀煌應該不是很甘願的辭職的，當初傳言蔡英文內定唐鳳為部長時，唐郭兩位之間可能稍微有點尷尬，後續唐鳳上任後在人事的溝通協調上也沒那麼順利。

**Reply by emily40533** [reply]
這篇信傳媒的文章雖然也是4月的報導，但我認為寫的蠻詳盡的
https://www.cmmedia.com.tw/home/articles/33272
還有這篇聯合新聞網的內容也可以參考一下~~
https://www.google.com/amp/s/udn.com/news/amp/story/6656/6360992
